src/cloud/apis/reporting.ts

```diff
@@ -51,3 +56,28 @@ export const reportPoints = (batch: PointsBatch) => {
     }
   }
 }
+
+const asRudderOptions = (tags: PointTags, campaign: object) => {
+  const options = {}
+  let context = null
+  Object.keys(tags).forEach((key: string) => {
+    const value = tags[key]
+    if (key != 'context') {
+      options[key] = value
+    } else {
+      if (typeof value != 'object') {
```

Comment:
```suggestion
      if (typeof value !== 'object') {
```

Also, do we care if the value is not an array?